bower.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "adv-fe-home-task",
+  "homepage": "https://github.com/AndreyWeber/adv-fe",
+  "authors": [
+    "Andrey Weber <Andrey.Weber@gmail.com>"
+  ],
+  "description": "Advanced Front-End cources home task project",
+  "main": "",
+  "license": "MIT",
+  "private": true,
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "underscore": "1.8.3",
+    "backbone": "^1.0.0",
+    "react": ">0.1.0",
+    "angular": "^1.5.5",
```

Comment:
> any version

please check documentation for "any version"
https://github.com/npm/node-semver